README.md

```diff
@@ -1,237 +1,14 @@
-# gravity-ui-web
+# Gravity UI
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/buildit/gravity-ui-web.svg)](https://greenkeeper.io/)
+This project is a [Lerna](https://lernajs.io/) monorepo composed of several sub-packages. Everything is binded
+together with a custom [Semantic Release](https://github.com/semantic-release/semantic-release) pipeline configuration.
 
-Library of styles, components and associated assets to build UIs for the web. Part of Buildit's Gravity design system.
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 
```

Comment:
I think it would be good to link to the packages that are within this monorepo. As this readme is likely to be the landing point still for most people and then they can click through to the correct sub-package (and readme) from here

Reply:
I agree. Just a bullet point list of links would suffice for now I think.

Reply:
Added a list of currently maintained packages